src/main.rs

```diff
@@ -1,29 +1,79 @@
-use std::hash::Hash;
-use std::io::ErrorKind;
-use std::iter::FromIterator;
-use std::{
-    collections::HashSet,
-    fs::{self, File},
-    process::Command,
-};
-
-use anyhow::{bail, Error, Result};
+mod handler;
+use anyhow::{bail, Result};
 use clap::{Parser, Subcommand, ValueEnum};
+use config::{Config, File, FileFormat};
 use glob::glob;
-use serde::{Deserialize, Serialize};
+use handler::*;
+use serde::Deserialize;
+use std::error::Error;
+use std::path::Path;
+use std::process::Command;
+use std::str;
+
+/// dir that greenboot looks for the health check and other scripts
+static GREENBOOT_INSTALL_PATHS: [&str; 2] = ["/usr/lib/greenboot", "/etc/greenboot"];
+
+/// greenboot config path
+static GREENBOOT_CONFIG_FILE: &str = "/etc/greenboot/greenboot.conf";
 
 #[derive(Parser)]
 #[clap(author, version, about, long_about = None)]
 #[clap(propagate_version = true)]
+
+/// cli parameters for greenboot
 struct Cli {
     #[clap(value_enum, short, long, default_value_t = LogLevel::Info)]
     log_level: LogLevel,
-
     #[clap(subcommand)]
     command: Commands,
 }
+#[derive(Debug, Deserialize)]
+///config params for greenboot
```

Comment:
this bothers me personally 😛 consistency with spaces after `///` - the code is a mix afaics